tests/test_config_parsing/config_dict_generator.py

```diff
@@ -333,15 +328,11 @@ def config_dicts(draw):
             }
         )
     )
-    should_exist_files = [
-        job[ConfigKeys.PATH] for job in config_dict[ConfigKeys.INSTALL_JOB]
-    ]
+    should_exist_files = [job[1] for job in config_dict[ConfigKeys.INSTALL_JOB]]
```

Comment:
I think this should be possible:
in place of:
```python
[job[1] for job in config_dict[ConfigKeys.INSTALL_JOB]]
```
to have 
```python
[ job_path for _, job_path in config_dict[ConfigKeys.INSTALL_JOB]]
```

Reply:
Fixed, and generally used that pattern throughout.